pkg/rancher-desktop/pages/Required.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
Same comment as in the translations file: the name `Required.vue` could be more descriptive.